mbs_results/inputs.py

```diff
@@ -0,0 +1,9 @@
+import json
+
+
+def load_config(path="config.json"):
```

Comment:
Function is missing doc strings.  Seems a pretty self explanatory function based on its name but might be worth adding quickly